synapse/models/infotech.py

```diff
@@ -995,6 +1000,13 @@ def getModelDefs(self):
                         'doc': 'The file that contains the function.'}),
                     ('va', ('int', {}), {
                         'doc': 'The virtual address of the first codeblock of the function'}),
+                    ('rank', ('int', {}), {
+                        'doc': 'The function rank score used to evaluate if it exhibits interesting behavior.'}),
+                    ('cmplx', ('int', {}), {
+                        'doc': 'The cyclomatic complexity of the function.'}),
```

Comment:
Let's avoid specifying that they use a specific complexity calculation and just allow it to be user-defined